decidim-meetings/app/models/decidim/meetings/meeting.rb

```diff
@@ -79,6 +88,21 @@ def can_participate?(user)
         return true if participatory_space.try(:private_space?) && participatory_space.users.include?(user)
         return false if participatory_space.try(:private_space?) && participatory_space.try(:is_transparent?)
       end
+
+      def can_view_meeting?(current_user)
```

Comment:
Isn't this logic the same as in the `current_user_can_visit_meeting?` method in the controller? If so, please use this method there

Reply:
after specs failing, I saw that the logic is not the same.
One method is to check if user can view the meeting, and the other one is to check if user can register to meeting.